src/tools/log-parser-rs/README.md

```diff
@@ -0,0 +1,61 @@
+# `kata-log-parser`
+
+## Introduction
+
+`kata-log-parser` is a tool that combines logfiles generated by the various
+system components, sorts them by timestamp, and re-displays the log entries. A
+time delta is added to show how much time has elapsed between each log entry.
+
+The tool is also able to check the validity of all log records, can re-format the
+logs, and output them in a different format.
+
+For more information on the `kata-log-parser` tool, use the help command:
+
+```
+$ kata-log-parser --help
+```
+
+## Log Format
+
+Kata runtime-rs logs are JSON objects in the following format:
+
+```
+{"msg":"message","level":"INFO","ts":"1970-01-01T00:00:00.000000000Z","name":"kata-runtime","version":"0.1.0","pid":"0","source":"source","subsystem":"subsystem"}
+```
+
+However, if `--ignore-missing-fields` is set, a log missing one or more of the following fields may be ommitted: `level`, `name`, `version`, `pid`, `source`, or `subsystem`.
+
+> **Note** a log entry must be on one single line, and a line must contain only one log entry.
+
+## Command line opts
```

Comment:
a compromise might be to list a few top, bare essential arguments and ask the user to use help to get the full up-to-date list.

Reply:
can say if -o not supplied it defaults to stdout

Reply:
I think ill go with mkbhanda's suggestion, as I personally get a bit irked when the only cli documentation for a tool is its --help flag, because sometimes I need to read up on it without having aces to the compiled tool itself. I can see why we dont want a full manpage, but the basics (--strict, --quiet, --output-file, etc) should be in there.